### DISCUSSION TITLE
Issue and more informations

### DISCUSSION THREAD

**Discussion by hpotier** [microsoft/azure-redcap-paas#81] [Q&A]
Hi @SvenAelterman 

I would like to deploy REDCap AZURE paas, and I've seen you've done a lot of modifications in past few months.
I try to follow your guide here : https://github.com/microsoft/azure-redcap-paas/blob/main/manual.md

Could you tell me more about all the service deployed? and specially in **Bold**

Virtual Network
**Key Vault ?????**
Storage Account
**Container Registry ?????**
PostgreSQL Flexible Server
**Container Instance ?????**
MySQL Flexible Server
Web App
App Service Plan
**Application Insights ?????**
**User-assigned Managed Identity ?????**
Deployment Script
**Log Analytics Workspace ?????**

Then, I don't really understant this parameter "identityObjectId" I've put '00000000-0000-0000-0000-000000000000' in bicep new file.
The same with "vnetAddressPrefix", i've keep your default value : '10.0.0.0/24'

is redcapZipUrl should be inside AZURE or can be a external website?

After a long time to deploy, I've encounter these issues:
![image](https://github.com/user-attachments/assets/96630913-32c6-4645-9fc5-c24a2c96c753)
Could you help to resolve them?

I've see there's so much more resources (and resources groups) than previously...
Could you explain me why, and If this new resources are "free" to use?

Could you also tell me, what kind of storage type do you use? I've found in "toClean" directory some "storageType" "Standard_LRS" but not in your new script. I would like to deploy a "Standard_ZRS" for this service.

How can we deploy a SendGrid service?

Sorry, If my questions seems to be obvious for you but it's not for me ;-)

Look forward to hearing from you,

Hugo

**Comment by hpotier** [comment]
I've finally install on AZURE from scratch!

**Comment by SvenAelterman** [comment]
@hpotier I am sorry for the lack of replies but I am glad to read that you were able to deploy REDCap on Azure. 

If you have specific feedback on how we can make the experience better for you, please feel free to share here.
